### PR TITLE
Play behavior fix (gpg)

### PR DESCRIPTION
**Pull request by cheqpat**
Managed to set up GPG commits and then got sidetracked between work and managing to scuff my branch in about fifteen  different ways. 
Anyway, I used udany's original code on the first commit, second commit just plays rather than alerting it being paused as mentioned in the original PR.
Let me know if anything needs to be changed from there.

### CODE REVIEW

**Comment by codetheweb** [comment]
(See https://github.com/codetheweb/muse/pull/325 and https://github.com/codetheweb/muse/issues/356 for context.)

Played with this a little and playback behavior seems better. My only comment would be that if:
1. You start playing a track
2. All users disconnect, as does Muse
3. You rejoin the voice channel and tell Muse to play a different track (i.e. `.p another track`)
4. Muse joins but doesn't resume streaming

In other words, Muse only resumes if the queue size is 0 currently. I think this should be changed so Muse always resumes.

**Comment by codetheweb** [comment]
Looks like I don't have permission to edit the files in this PR so I opened a new one: https://github.com/codetheweb/muse/pull/424
